docs/product-user-guide/src/main/asciidoc/chap-red-hat-jboss-dashboard-builder.adoc

```diff
@@ -1,83 +1,173 @@
 [id='_chap_red_hat_jboss_dashboard_builder']
 = Red Hat JBoss Dashboard Builder
 
-Red Hat JBoss Dashboard Builder
- is a web-based dashboard application that provides Business Activity Monitoring (BAM) support, that is, visualization tools for monitored metrics (Key Performance Indicators, or KPIs) in real time.
-It comes integrated in the Business Central environment under the *Dashboards* menu.
+Red Hat JBoss Dashboard Builder application supports Business Activity Monitoring (BAM) using visualization tools for monitoring metrics, such as Key Performance Indicators and process statuses. With the Dashboard Builder, you can compose custom business dashboards by mixing data coming from various data sources and displaying them as graphs or tables, as well as creating custom title navigation.
```

Comment:
I wouldn't call it *application*, as it's no longer separate web application, but rather integral component of the workbench. How about replacing that work with *component*?

Also I never heard the term "title navigation" before - maybe just remove the word "title" if such thing doesn't exist or ignore it if there is such a thing :)



Dashboard Builder component is an integral parf

Reply:
Not sure what the last sentence here means(typo?), but I've changed the other two words you mentioned.

Reply:
Sorry, my comments are full of typos :)
I probably wanted to delete the last sentence as I edited the comment several times over. It was meant to be "Dashboard Builder component is an integral part of the workbench"